lib/shared/validation/schemas/index.ts

```diff
@@ -1,2 +1,4 @@
 export { OptionalPhoneValidationSchema } from './OptionalPhoneValidationSchema'
 export { RequiredPhoneValidationSchema } from './RequiredPhoneValidationSchema'
+export { E164PhoneValidationSchema } from './E164PhoneValidationSchema'
```

Comment:
I think the validation schema you wrote makes `OptionalPhoneValidationSchema` and `RequiredPhoneValidationSchema` obsolete, no? Because I see you can actually chain zod functions like `E164PhoneValidationSchema.optional()` to mark that a phone field is optional.

If so, I would remove the other functions and use the one you created in the places we used them (luckily, there are not a lot of those).

Reply:
I've seen those two validators you mentioned, but they don't use that rigorous validation. See https://github.com/catamphetamine/libphonenumber-js#validate-phone-number
> isPossiblePhoneNumber() only validates phone number length, while isValidPhoneNumber() validates both phone number length and the actual phone number digits.

I didn't alter those validators, because I don't want to break any stuff. If you think it is safe to replace, please confirm it. I will change that accordingly.

Reply:
Yup, you should be good (and the tests `yarn test` will confirm).